### PR TITLE
[14.0] [ADD] project_reference

### PR DESCRIPTION
**Pull request by watthanun**

### CODE REVIEW

File: oca.cfg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[GitHub]
```
Comment (Saran440) [inline oca.cfg]: what is this file?

File: project_reference/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Reference",
+    "summary": "Add a reference to projects",
+    "version": "14.0.1.0.0",
+    "category": "Project",
+    "website": "https://github.com/OCA/project",
+    "author": "Watthanun Khorchai," "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline project_reference/__manifest__.py]:
```suggestion
    "author": "Watthanun Khorchai, Odoo Community Association (OCA)",
```

File: project_reference/models/project_task.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2019-2020 Onestein
```
Comment (Saran440) [inline project_reference/models/project_task.py]:
```suggestion
```

Add copyright with your company or empty

File: project_reference/views/project_task_view.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019-2020 Onestein
```
Comment (Saran440) [inline project_reference/views/project_task_view.xml]:
```suggestion
<!-- Copyright 2021 xxxx
```
add your company or empty

**Review by Saran440 — COMMENTED** [review]
Code Reviewed.

File: project_reference/README.rst [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+.. |badge3| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_reference
+    :alt: Translate me on Weblate
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/14.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+This module extends the functionality of projects to support a reference for each project
```
Comment (lfreeke) [inline project_reference/README.rst]: It seems that now the reference is shown on the project task.  

I would propose this as sentence: 
This module extends the functionality of projects to support a reference for each project task

**Review by flotho — COMMENTED** [review]
Hi,
Could you precise the use case of this module. 
Is it a "duplicate of project_task_code in term of use case?"
Regarding the code the module name should be project_task_reference

Thanks

**Pull request closed without merging**